config/application.rb

```diff
@@ -26,5 +26,6 @@ class Application < Rails::Application
     # Middleware like session, flash, cookies can be added back manually.
     # Skip views, helpers and assets when generating a new resource.
     config.api_only = true
+    config.autoload_paths << Rails.root.join('lib')
```

Comment:
You *might* have to do `+= [Rails.root.join('lib')]` .  Check how ManageIQ does it.  I recall there being a weird side effect if you don't.  cc @jrafanie

Reply:
@Fryguy It uses the shovel operator 
https://github.com/ManageIQ/manageiq/blob/0aae68f51a4d549cc74e592b1b8fd2818bbe2da9/config/application.rb#L108
I am missing the to_s at the end I will fix that

Reply:
This looks ok to me.  I vaguely recall what @Fryguy is mentioning but don't remember the details.  My git fu doesn't show anything besides 90197346f0fc35e7827dbf6199627473ef19c1fc commit which converted `+=` to `<<`.  There wasn't a reason mentioned.  I'm guessing we just wanted to remove the unnecessary array creation by using the shovel operator.  Perhaps, at one time, the autoload paths were frozen so you had to duplicate them and make changes but that is clearly not the case now.